mantidproject/mantid
# Indirect diffraction bug fixes

The Indirect Diffraction interface now correctly performs a rebin in d-spacing when in diffspec mode

**To test:**

Requires access to ISIS data archive

Interfaces>Indirect>Diffraction

Instrument: IRIS
Mode: diffspec

Run: 26176
Vanadium: 26173

Fill in some values for rebin eg `3.0` `0.01` `5.0`

press `Run`

ensure resulting workspace has been rebinned in d-spacing as specified.

Fixes #19610 .

*Does not need to be in the release notes.*


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
